.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,39 @@
+---
+labels: Needs Review
+name: Pull Request
+---
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+- [ ] Desktop screenshots
+- [ ] Mobile width screenshots
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Enhancement (improves on existing features/functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Version/configuration (updating dependency versions or modifying configuration settings)
```

Comment:
TBH, I find this list useless. It requires a lot of cognitive overhead, and I don't know how it changes anything in the review process.